nodejs/tfwconnector/TWFConnector.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const zmq = require('zeromq'),
+    subSock = zmq.socket('sub'),
+    pushSock = zmq.socket('push');
+
+const SUB_PORT = process.env.TFW_PUBLISHER_PORT || 7654,
+    PUSH_PORT = process.env.TFW_RECEIVER_PORT || 8765;
+
+const SUB_ADDRESS = "tcp://localhost:" + SUB_PORT,
+      PUSH_ADDRESS = "tcp://localhost:" + PUSH_PORT;
+
+
+class TFWConnector {
```

Comment:
This should be named `TFWServerConnector` to harmonize with python, cpp and java implementations.

Reply:
The `MessageSender.send() and MessageSender.sendToEventHandlers()` methods should be implemented here as mentioned above (`MessageSender` is only meant to write instructions).